selfdrive/car/gm/carstate.py

```diff
@@ -35,7 +36,7 @@ def update(self, pt_cp, loopback_cp):
     ret.brakePressed = pt_cp.vl["EBCMBrakePedalPosition"]["BrakePedalPosition"] >= 10
 
     # Regen braking is braking
-    if self.car_fingerprint == CAR.VOLT:
+    if self.CP.transmissionType == TransmissionType.direct:
```

Comment:
Two things.
1. The Bolt EUV is only the first new electric car - rather than trying to hardcode I thought it prudent to use a property of some sort
2. I originally used a set defined in values.py called EV_CAR and included the cars; then I would check if the fingerprint was in the collection. This was before I had even a basic understanding of cereal / capnp.
After noticing that CarParams already contains an attribute to differentiate between EV and Gas cars, I made an assumption - based on the presence of these fields in the CarParams - that we should use them.

If this assumption was wrong, I can redo it using the static values